src/Symfony/Component/Messenger/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 -----
 
  * `InMemoryTransport` can perform message serialization through dsn `in-memory://?serialize=true`.
+ * Deprecated `prefetch_count` parameter, it has no effect and will be removed in Symfony 6.0.
```

Comment:
This one should go in the amqp bridge's changelog I think

Reply:
right